paper/paper.bib

```diff
@@ -0,0 +1,82 @@
+@article{Trounstine:2009,
+  issn      = {15375927, 15410986},
+  url       = {http://www.jstor.org/stable/40407007},
+  abstract  = {The study of local politics has been relegated to the periphery of political science and many explanations have been offered for the marginalization of the subfield. I offer three related arguments for why scholars should revisit the study of sub-state politics. First, the local level is the source of numerous political outcomes that matter because they represent a large proportion of political events in the United States. Secondly, there are methodological advantages to studying local politics. Finally, analyzing politics at the sub-state level can generate thoroughly different kinds of questions than a purely national-level focus and can offer different answers to questions that apply more generally. Research on local politics can and should contribute to broader debates in political science and ensure that we understand both how and why cities are unique.},
+  author    = {Jessica Trounstine},
+  journal   = {Perspectives on Politics},
+  number    = {3},
+  pages     = {611--618},
+  publisher = {[American Political Science Association, Cambridge University Press]},
+  title     = {All Politics Is Local: The Reemergence of the Study of City Politics},
+  volume    = {7},
+  year      = {2009}
+}
+
+@misc{cookiecutter,
+  author    = {Audrey Roy Greenfeld and Daniel Roy Greenfeld and Raphael Pierzina},
+  title     = {cookiecutter},
+  abstract  = {A command-line utility that creates projects from cookiecutters (project templates), e.g. creating a Python package project from a Python package project template.},
+  year      = {2015},
+  publisher = {GitHub},
+  journal   = {GitHub repository},
+  url       = {https://github.com/cookiecutter/cookiecutter}
+}
+
+@article{Sparks2017,
+  doi       = {10.21105/joss.00411},
+  url       = {https://doi.org/10.21105/joss.00411},
+  year      = {2017},
+  publisher = {The Open Journal},
+  volume    = {2},
+  number    = {17},
+  pages     = {411},
+  author    = {Adam H. Sparks and Mark Padgham and Hugh Parsonage and Keith Pembleton},
+  title     = {bomrang: Fetch Australian Government Bureau of Meteorology Data in R},
+  journal   = {Journal of Open Source Software}
+}
+
+@misc{courtlistener,
+  author    = {{The Free Law Project}},
```

Comment:
How you want it rendered in the paper. Without the extra set it will be rendered as `(Project, 2015)` in the inline citation. The extra brackets renders it as `(The Free Law Project, 2015)`

Reply:
Ooh TIL, thanks for the clarification.